packages/flutter/lib/src/animation/animation.dart

```diff
@@ -60,33 +60,93 @@ typedef AnimationStatusListener = void Function(AnimationStatus status);
 /// Signature for method used to transform values in [Animation.fromValueListenable].
 typedef ValueListenableTransformer<T> = T Function(T);
 
-/// An animation with a value of type `T`.
+/// A value which may change over time, moving forward or backward.
 ///
-/// An animation consists of a value (of type `T`) together with a status. The
-/// status indicates whether the animation is conceptually running from
-/// beginning to end or from the end back to the beginning, although the actual
-/// value of the animation might not change monotonically (e.g., if the
-/// animation uses a curve that bounces).
+/// An animation has a [value] (of type [T]) and a [status].
+/// The value conceptually lies on some path, and
+/// the status indicates how the value is currently moving along the path:
+/// forward, backward, or stopped at the end or the beginning.
+/// The path may double back on itself
+/// (e.g., if the animation uses a curve that bounces),
+/// so even when the animation is conceptually moving forward
+/// the value might not change monotonically.
 ///
-/// Animations also let other objects listen for changes to either their value
-/// or their status. These callbacks are called during the "animation" phase of
+/// Consumers of the animation can listen for changes to either the value
+/// or the status, with [addListener] and [addStatusListener].
+/// The listener callbacks are called during the "animation" phase of
 /// the pipeline, just prior to rebuilding widgets.
 ///
-/// To create a new animation that you can run forward and backward, consider
-/// using [AnimationController].
+/// An animation may move forward or backward on its own as time passes
+/// (like the opacity of a button that fades over a fixed duration
+/// once the user touches it),
+/// or it may be driven by the user
+/// (like the position of a slider that the user can drag back and forth),
+/// or it may do both
+/// (like a switch that snaps into place when releaseed,
+/// or a [Dismissible] that responds to drag and fling gestures, etc.).
+/// The behavior is normally controlled by method calls on
+/// some underlying [AnimationController],
+/// for example by a [State] that created the [AnimationController].
```

Comment:
Not a huge deal, but I did have to re-read this a couple of times, since at first glance it sounded like a `State` is an example of an `AnimationController`.

Perhaps it could just be deleted, since it's also explained [further down](https://github.com/flutter/flutter/blob/c15bf8ead6d07dfa1755fba41c1c8ed0c81a50f4/packages/flutter/lib/src/animation/animation.dart#L103-L107).

```suggestion
```